index.html

```diff
@@ -99,8 +99,8 @@ <h2>I Love Coding.</h2>
 								<span id="typed"></span>
 							</div>
 							<div>
-								<p>If you need a Website, then <a href="mailto:gauravgupta14376@gmail.com">email</a> me
-									may be.</p>
+								<p>If you want such an amazing website, then contact by an<a href="mailto:gauravgupta14376@gmail.com">Email</a>
```

Comment:
**If you want such an amazing website, then contact by anEmail**
It will look like this, it has some errors

Reply:
Sorry..i corrected plz checkout